decaf377-ka/src/lib.rs

```diff
@@ -27,6 +28,8 @@ pub struct SharedSecret(pub [u8; 32]);
 pub enum Error {
     #[error("Invalid public key")]
     InvalidPublic(Public),
+    #[error("Cannot deserialize public key")]
+    PublicKeyDeserializationError,
```

Comment:
Could this just be `SliceLenError`, since that's the only failure case of `TryFrom`?